README.md

```diff
@@ -15,6 +21,16 @@ Included are the following:
 - testscript: script-based testing based on txtar files
 - txtar: simple text-based file archives for testing.
 
-# Links
+### testscript
 
-- [Test scripts in Go](https://bitfieldconsulting.com/golang/test-scripts)
+The most popular package here is the [testscript](https://pkg.go.dev/github.com/rogpeppe/go-internal/testscript) package:
+ * Provides a shell-like test environment that is very nicely tuned for testing Go CLI commands.
+ * Extracted version of the core Go team's internal testscript package.
+ * [Heavily used](https://github.com/golang/go/tree/master/src/cmd/go/testdata/script) to test the `go` command.
```

Comment:
Having this comment as a separate bullet seems to suggest that `github.com/rogpeppe/go-internal/testscript` is used in the `cmd/go` tests, which it is not. I would perhaps fix that by connecting this bullet to the previous one, and adjusting the wording to "link" such that it is clear the internal "version" of `testscript` is used for `cmd/go` tests. Perhaps even linking to and naming https://github.com/golang/go/tree/master/src/cmd/go/internal/script.

Reply:
Good point. 

(I originally wrote these bullets as a paragraph with more connective words, but then thought bullets would be better, and in the process created the problem you point out).